README.md

```diff
@@ -1,4 +1,4 @@
-# REDCap module: Image Map
+w# REDCap module: Image Map
```

Comment:
@123andy, I'm going to merge your PR, but this line looks like a typo. I'm going to revert it after the merge.